.github/workflows/kind.yml

```diff
@@ -17,6 +17,7 @@ jobs:
   kind-tests:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
Why set `fail-fast: false`?

Reply:
It's nice to see if *any* of the matrix instances (or whatever they're called) pass. I added it when I had an issue that only happened in the hosted mode tests, and I wanted to know if the non-hosted mode tests were affected.

Reply:
That makes sense. Maybe we should set that for all our repos...